docker/Dockerfile

```diff
@@ -23,7 +23,7 @@ ARG DEBIAN_FRONTEND=noninteractive
 ARG GCC_VERSION=11
 ARG LIGHTNING_VERSION=v0.36.0
 ARG PENNYLANE_VERSION=v0.36.0
-ARG ROCM_INSTALLER=https://repo.radeon.com/amdgpu-install/6.1.1/ubuntu/jammy/amdgpu-install_6.1.60101-1_all.deb
+ARG ROCM_INSTALLER=https://repo.radeon.com/amdgpu-install/5.7/ubuntu/jammy/amdgpu-install_5.7.50700-1_all.deb
```

Comment:
Is there any reason to downgrade rocm version?

Reply:
Didn't match the image version [here](https://github.com/PennyLaneAI/pennylane-lightning/blob/be9459f489d32d0624c37c71730e7f92379f5ba4/docker/Dockerfile#L196) with which it was tested on LUMI.